.github/workflows/npm-cd.yml

```diff
@@ -92,7 +92,8 @@ jobs:
               shell: bash
               working-directory: ./node
               run: |
-                  npm publish
+                  npm set "//registry.npmjs.org//:_authToken=$NODE_AUTH_TOKEN"
```

Comment:
I'm almost sure it's redundant, it is being set using the publishConfig

Reply:
where is it set?
IIRC the publish didn't work until I added this.

Reply:
I tested it yesterday, it works without this line. We're passing the auth token through the NODE_AUTH_TOKEN and the repository url through publishConfig

Reply:
remoed